DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -47,7 +47,7 @@
 
 ## IntelliJ
 - Install the latest Ultimate edition IntelliJ
-- Set the project SDK to use the installed Zulu Java 8 SDK
+- Set the project SDK to use the installed Zulu Java 11 SDK
```

Comment:
Remove the version totally as with Eclipse. The Java version to use should be singularly mentioned in paragraph "Java" to avoid such mistakes